pkg/config/config.go

```diff
@@ -210,6 +212,28 @@ func NewCreateChannelTx(channelConfig Channel, channelID string) (*cb.Envelope,
 	return env, nil
 }
 
+// NewSystemChannelBlock creates a genesis block using the provided consortiums and orderer
+// configuration and returns a block.
+func NewSystemChannelBlock(channelConfig Channel, channelID string) (*cb.Block, error) {
+	var err error
+
+	if channelID == "" {
+		return nil, errors.New("system channel ID is required")
+	}
+
+	ct, err := newSystemChannelGroup(channelConfig)
```

Comment:
you may want to update this function and the `newChannelGroup` function for application channels. Application channels must have the `channelConfig.Consortium` specified whereas system channels do not (looks like this behavior on master is currently swapped). `newSystemChannelGroup` and `newChannelGroup` are also both missing a modPolicy: `channelGroup.ModPolicy = channelConfig.AdminsPolicyKey`

Reply:
I fixed the two problems in fucntion newSystemChannelGroup and newChannelGroup. Please review whether it is correct.